src/encoded/static/components/filegallery.js

```diff
@@ -2500,40 +2571,20 @@ const AnalysesSelector = ({ analyses, selectedAnalysesIndex, handleAnalysesSelec
     });
 
     /**
-     * Selects certain awards as the default if listed.
+     * Select default for "Choose Analysis".
     */
     useMount(() => {
-        const awardSuffix = ['ENCODE4', 'ENCODE3', 'ENCODE2', 'Mixed', 'Lab custom']; // ordered in decreasing precedence
-        const analysesAwarded = analyses.filter((analysis) => awardSuffix.some((award) => (analysis.title || '').includes(award)));
-
-        // a bit hacky, removes text and use number to determine sorting
-        const sortedAnalysesAwarded = analysesAwarded.sort((a, b) => {
-            // remove text
-            const numbersInTitle1 = a.title.replace(/\D/g, '');
-            const numbersInTitle2 = b.title.replace(/\D/g, '');
+        const awardSuffixes = ['ENCODE4', 'ENCODE3', 'ENCODE2', 'Mixed', 'Lab custom']; // ordered in decreasing precedence
+        const statuses = ['released', 'in progress', 'archived', 'revoked', 'deleted']; // ordered in decreasing precedence
```

Comment:
You can get the statuses of any defined object type in a certain order in `getObjectStatuses`, and filtered to only include those available at the user’s current access level. You’d have to add an entry for “Analysis” in `defaultObjectStatuses` as we update this object as needed.

Reply:
@forresttanaka I tried this. It seems to only give user's current access level. It does not always correlate o the analysis_object status. Like in **<url>//experiments/ENCSR000EIY/**, admin has access levels "released”, “in progress”, “deleted". But analysis_object has these and "archived"

Reply:
`getObjectStatuses` takes the user’s current access level in the `accessLevel` parameter — it doesn’t return the user’s current access level. It “Returns array of possible statuses for given object.” For example, if you pass `getObjectStatuses` an “item” of “AntibodyCharacterization” and `accessLevel` “external,” it returns the array `["compliant", "exempt from standards", "not compliant", "not reviewed", "not submitted for review by lab"]`.